backend/benefit/pyproject.toml

```diff
@@ -3,5 +3,6 @@ extend-exclude = '''
 (
   .pytest_cache
   | var/media
+  | migrations
```

Comment:
It would be good to keep these pyproject.toml and setup.cfg files identical within the different projects inside the yjdh monorepo IMHO. Best would be that there wouldn't be any separate files for each project but they would all use the same files. I synced the files at some point, now doing this will make them different again.

Reply:
Yes, this could be added to other pyproject.toml's too. The reason I had to add it in the first place was that older black versions ignored migrations by default. Now that Django has made the decision that Django's autogenerated migrations files all are formatted by black by default from Django version 5 onwards, black no longer ignores them.